packages/backend/src/database/migrations/20220428071116_job_steps.ts

```diff
@@ -0,0 +1,42 @@
+import { Knex } from 'knex';
+
+const COMPILE_JOB_STEPS_TABLE_NAME = 'job_steps';
+const COMPILE_JOB_TABLE_NAME = 'jobs';
+
+export async function up(knex: Knex): Promise<void> {
+    // https://github.com/knex/knex/pull/4657
+    await knex.schema.alterTable(COMPILE_JOB_TABLE_NAME, (tableBuilder) => {
+        tableBuilder.uuid('project_uuid').nullable().alter();
```

Comment:
project_uuid needs to be nullable now, because we start job before project exists